grpc/metrics.go

```diff
@@ -0,0 +1,30 @@
+package grpc
+
+import "go.k6.io/k6/metrics"
+
+// grpcMetrics contains the metrics for the grpc extension.
+type grpcMetrics struct {
+	GRPCStreams                 *metrics.Metric
+	GRPCStreamsMessagesSent     *metrics.Metric
+	GRPCStreamsMessagesReceived *metrics.Metric
+}
```

Comment:
```suggestion
// streamsMetrics contains the metrics for the grpc extension.
type streamsMetrics struct {
	Streams                 *metrics.Metric
	MessagesSent     *metrics.Metric
	MessagesReceived *metrics.Metric
}
```
I'm for removing the redundancy.

Reply:
It could contain the module metrics, not only the stream's

Reply:
Oh, that's good info then I think I would like `instanceMetrics`. Anyway, the name of the struct is minor, I would prefer instead to drop the redundancy from the fields.

```suggestion
type instanceMetrics struct {
	Streams                 *metrics.Metric
	StreamsMessagesSent     *metrics.Metric
	StreamsMessagesReceived *metrics.Metric
}
```

Reply:
It sounds we haven't removed the GRPC redundancy from the fields